### ISSUE TITLE
Octave (4.2.0) gives an error after execution

### ISSUE THREAD

**Issue by skilldeliver** [judge0/api#104]
![image](https://user-images.githubusercontent.com/37806520/65095430-cfd4e280-d9c9-11e9-9be9-6835ab03b35a.png)

**Comment by hermanzdosilovic** [comment]
This has been resolved in `feature/compiler_options` branch where flag [`--no-history`](https://github.com/judge0/api/blob/feature%2Fcompiler_options/db/seeds.rb#L213) was added.

I will close this issue once this branch is merged into master.
